data/category.go

```diff
@@ -116,17 +119,24 @@ func (c *Category) Seed(cfg write.ClientConfig) (int, error) {
 	}
 }
 
-func (c *Category) Test(cfg write.ClientConfig) error {
+func (c *Category) Test(cfg write.ClientConfig) (err error) {
```

Comment:
This API is hard to follow, and I think it's in part due to lack of GoDoc.

This method assigns to the err variable but always returns nil.  Is the error supposed to indicate that something went legitimately wrong, or is supposed to indicate a test failure?